examples/wasm-shell/components/wasm-terminal.tsx

```diff
@@ -5,62 +5,18 @@ import WasmTerminal, {
   // @ts-ignore
   fetchCommandFromWAPM
   // @ts-ignore
-} from "../../../packages/wasm-terminal/dist/optimized/wasm-terminal.esm";
+} from "@wasmer/wasm-terminal";
 
+import WASI from "@wasmer/wasi";
+import BrowserWASIBindings from "@wasmer/wasi/bindings/browser";
```

Comment:
How come you need to do this? 

`WASI.defaultBindings` should already exist? 🤔 

Try logging the `WASI` object with `console.log(WASI)`, they should already be in there somewhere? 😮

Reply:
Because is not using the bundled version of rollup, we need to manually bind it.

Reply:
Makes sense! 😄 👍